2013-06-17 14:18:29 Error: Fatal Error (1): Call to a member function read() on a non-object in [/Users/jace/dev/dogsquare_web/app/View/Elements/header.ctp, line 2]
2013-06-17 14:18:29 Error: [FatalErrorException] Call to a member function read() on a non-object
Request URL: /
Stack Trace:
#0 /Users/jace/dev/dogsquare_web/lib/Cake/Error/ErrorHandler.php(184): ErrorHandler::handleFatalError(1, 'Call to a membe...', '/Users/jace/dev...', 2)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', '/Users/jace/dev...', 2, Array)
#2 /Users/jace/dev/dogsquare_web/lib/Cake/Core/App.php(927): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', '/Users/jace/dev...', 2, Array)
#3 /Users/jace/dev/dogsquare_web/lib/Cake/Core/App.php(900): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-06-17 14:45:10 Error: [MissingControllerException] Controller class UsersController could not be found.
Exception Attributes: array (
  'class' => 'UsersController',
  'plugin' => NULL,
)
Request URL: /users
Stack Trace:
#0 /Users/jace/dev/dogsquare_web/app/webroot/index.php(110): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-06-17 19:13:58 Error: [MissingComponentException] Component class CaptchaComponent could not be found.
Exception Attributes: array (
  'class' => 'CaptchaComponent',
  'plugin' => false,
)
Request URL: /users/login
Stack Trace:
#0 /Users/jace/dev/dogsquare_web/lib/Cake/Controller/ComponentCollection.php(53): ComponentCollection->load('Captcha', Array)
#1 /Users/jace/dev/dogsquare_web/lib/Cake/Controller/Controller.php(639): ComponentCollection->init(Object(UsersController))
#2 /Users/jace/dev/dogsquare_web/lib/Cake/Routing/Dispatcher.php(183): Controller->constructClasses()
#3 /Users/jace/dev/dogsquare_web/lib/Cake/Routing/Dispatcher.php(162): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest), Object(CakeResponse))
#4 /Users/jace/dev/dogsquare_web/app/webroot/index.php(110): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2013-06-17 19:15:25 Error: [MissingComponentException] Component class CaptchaComponent could not be found.
Exception Attributes: array (
  'class' => 'CaptchaComponent',
  'plugin' => false,
)
Request URL: /users/login
Stack Trace:
#0 /Users/jace/dev/dogsquare_web/lib/Cake/Controller/ComponentCollection.php(53): ComponentCollection->load('Captcha', Array)
#1 /Users/jace/dev/dogsquare_web/lib/Cake/Controller/Controller.php(639): ComponentCollection->init(Object(UsersController))
#2 /Users/jace/dev/dogsquare_web/lib/Cake/Routing/Dispatcher.php(183): Controller->constructClasses()
#3 /Users/jace/dev/dogsquare_web/lib/Cake/Routing/Dispatcher.php(162): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest), Object(CakeResponse))
#4 /Users/jace/dev/dogsquare_web/app/webroot/index.php(110): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2013-06-17 19:15:48 Error: Fatal Error (1): Call to a member function url() on a non-object in [/Users/jace/dev/dogsquare_web/app/View/Users/login.ctp, line 85]
2013-06-17 19:15:48 Error: [FatalErrorException] Call to a member function url() on a non-object
Request URL: /users/login
Stack Trace:
#0 /Users/jace/dev/dogsquare_web/lib/Cake/Error/ErrorHandler.php(184): ErrorHandler::handleFatalError(1, 'Call to a membe...', '/Users/jace/dev...', 85)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', '/Users/jace/dev...', 85, Array)
#2 /Users/jace/dev/dogsquare_web/lib/Cake/Core/App.php(927): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', '/Users/jace/dev...', 85, Array)
#3 /Users/jace/dev/dogsquare_web/lib/Cake/Core/App.php(900): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-06-17 19:25:15 Error: Fatal Error (1): Call to a member function url() on a non-object in [/Users/jace/dev/dogsquare_web/app/View/Users/login.ctp, line 85]
2013-06-17 19:25:15 Error: [FatalErrorException] Call to a member function url() on a non-object
Request URL: /users/login
Stack Trace:
#0 /Users/jace/dev/dogsquare_web/lib/Cake/Error/ErrorHandler.php(184): ErrorHandler::handleFatalError(1, 'Call to a membe...', '/Users/jace/dev...', 85)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', '/Users/jace/dev...', 85, Array)
#2 /Users/jace/dev/dogsquare_web/lib/Cake/Core/App.php(927): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', '/Users/jace/dev...', 85, Array)
#3 /Users/jace/dev/dogsquare_web/lib/Cake/Core/App.php(900): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-06-17 19:25:46 Error: [MissingLayoutException] Layout file "/Users/jace/dev/dogsquare_web/app/View/Layouts/blank.ctp" is missing.
Exception Attributes: array (
  'file' => '/Users/jace/dev/dogsquare_web/app/View/Layouts/blank.ctp',
)
Request URL: /users/login
Stack Trace:
#0 /Users/jace/dev/dogsquare_web/lib/Cake/View/View.php(509): View->_getLayoutFileName('blank')
#1 /Users/jace/dev/dogsquare_web/lib/Cake/View/View.php(479): View->renderLayout('', 'blank')
#2 /Users/jace/dev/dogsquare_web/lib/Cake/Controller/Controller.php(948): View->render(NULL, NULL)
#3 /Users/jace/dev/dogsquare_web/lib/Cake/Routing/Dispatcher.php(194): Controller->render()
#4 /Users/jace/dev/dogsquare_web/lib/Cake/Routing/Dispatcher.php(162): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest), Object(CakeResponse))
#5 /Users/jace/dev/dogsquare_web/app/webroot/index.php(110): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#6 {main}
2013-06-17 19:34:01 Error: [MissingViewException] View file "/Users/jace/dev/dogsquare_web/app/View/Users/reset_password.ctp" is missing.
Exception Attributes: array (
  'file' => '/Users/jace/dev/dogsquare_web/app/View/Users/reset_password.ctp',
)
Request URL: /users/resetPassword
Stack Trace:
#0 /Users/jace/dev/dogsquare_web/lib/Cake/View/View.php(468): View->_getViewFileName(NULL)
#1 /Users/jace/dev/dogsquare_web/lib/Cake/Controller/Controller.php(948): View->render(NULL, NULL)
#2 /Users/jace/dev/dogsquare_web/lib/Cake/Routing/Dispatcher.php(194): Controller->render()
#3 /Users/jace/dev/dogsquare_web/lib/Cake/Routing/Dispatcher.php(162): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest), Object(CakeResponse))
#4 /Users/jace/dev/dogsquare_web/app/webroot/index.php(110): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2013-06-17 21:36:20 Error: [MissingTableException] Table users for model User was not found in datasource default.
Exception Attributes: array (
  'table' => 'users',
  'class' => 'User',
  'ds' => 'default',
)
Request URL: /users/login
Stack Trace:
#0 /Users/jace/dev/dogsquare_web/lib/Cake/Model/Model.php(3243): Model->setSource('users')
#1 /Users/jace/dev/dogsquare_web/lib/Cake/Model/Model.php(800): Model->getDataSource()
#2 [internal function]: Model->__call('findAllByEmail', Array)
#3 /Users/jace/dev/dogsquare_web/app/Model/User.php(52): User->findAllByEmail('te')
#4 /Users/jace/dev/dogsquare_web/app/Controller/UsersController.php(98): User->validateAdminCredentials('te', 'te')
#5 [internal function]: UsersController->login()
#6 /Users/jace/dev/dogsquare_web/lib/Cake/Controller/Controller.php(486): ReflectionMethod->invokeArgs(Object(UsersController), Array)
#7 /Users/jace/dev/dogsquare_web/lib/Cake/Routing/Dispatcher.php(187): Controller->invokeAction(Object(CakeRequest))
#8 /Users/jace/dev/dogsquare_web/lib/Cake/Routing/Dispatcher.php(162): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest), Object(CakeResponse))
#9 /Users/jace/dev/dogsquare_web/app/webroot/index.php(110): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#10 {main}
2013-06-17 21:37:14 Error: [PDOException] SQLSTATE[42S22]: Column not found: 1054 Unknown column 'User.email' in 'where clause'
Request URL: /users/login
Stack Trace:
#0 /Users/jace/dev/dogsquare_web/lib/Cake/Model/Datasource/DboSource.php(460): PDOStatement->execute(Array)
#1 /Users/jace/dev/dogsquare_web/lib/Cake/Model/Datasource/DboSource.php(426): DboSource->_execute('SELECT `User`.`...', Array)
#2 /Users/jace/dev/dogsquare_web/lib/Cake/Model/Datasource/DboSource.php(669): DboSource->execute('SELECT `User`.`...', Array, Array)
#3 /Users/jace/dev/dogsquare_web/lib/Cake/Model/Datasource/DboSource.php(1080): DboSource->fetchAll('SELECT `User`.`...', false)
#4 /Users/jace/dev/dogsquare_web/lib/Cake/Model/Model.php(2696): DboSource->read(Object(User), Array)
#5 /Users/jace/dev/dogsquare_web/lib/Cake/Model/Datasource/DboSource.php(594): Model->find('all', Array)
#6 /Users/jace/dev/dogsquare_web/lib/Cake/Model/Model.php(800): DboSource->query('findAllByEmail', Array, Object(User))
#7 [internal function]: Model->__call('findAllByEmail', Array)
#8 /Users/jace/dev/dogsquare_web/app/Model/User.php(52): User->findAllByEmail('te')
#9 /Users/jace/dev/dogsquare_web/app/Controller/UsersController.php(98): User->validateAdminCredentials('te', 'te')
#10 [internal function]: UsersController->login()
#11 /Users/jace/dev/dogsquare_web/lib/Cake/Controller/Controller.php(486): ReflectionMethod->invokeArgs(Object(UsersController), Array)
#12 /Users/jace/dev/dogsquare_web/lib/Cake/Routing/Dispatcher.php(187): Controller->invokeAction(Object(CakeRequest))
#13 /Users/jace/dev/dogsquare_web/lib/Cake/Routing/Dispatcher.php(162): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest), Object(CakeResponse))
#14 /Users/jace/dev/dogsquare_web/app/webroot/index.php(110): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#15 {main}
2013-06-17 21:47:59 Error: Fatal Error (1): Call to undefined method CookieComponent::del() in [/Users/jace/dev/dogsquare_web/app/Controller/UsersController.php, line 118]
2013-06-17 21:47:59 Error: [FatalErrorException] Call to undefined method CookieComponent::del()
Request URL: /users/login
Stack Trace:
#0 /Users/jace/dev/dogsquare_web/lib/Cake/Error/ErrorHandler.php(184): ErrorHandler::handleFatalError(1, 'Call to undefin...', '/Users/jace/dev...', 118)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to undefin...', '/Users/jace/dev...', 118, Array)
#2 /Users/jace/dev/dogsquare_web/lib/Cake/Core/App.php(927): call_user_func('ErrorHandler::h...', 1, 'Call to undefin...', '/Users/jace/dev...', 118, Array)
#3 /Users/jace/dev/dogsquare_web/lib/Cake/Core/App.php(900): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-06-18 07:19:18 Error: Fatal Error (1): Call to undefined method UsersController::requireLogin() in [/Users/jace/dev/dogsquare_web/app/Controller/UsersController.php, line 21]
2013-06-18 07:19:18 Error: [FatalErrorException] Call to undefined method UsersController::requireLogin()
Request URL: /users/index
Stack Trace:
#0 /Users/jace/dev/dogsquare_web/lib/Cake/Error/ErrorHandler.php(184): ErrorHandler::handleFatalError(1, 'Call to undefin...', '/Users/jace/dev...', 21)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to undefin...', '/Users/jace/dev...', 21, Array)
#2 /Users/jace/dev/dogsquare_web/lib/Cake/Core/App.php(927): call_user_func('ErrorHandler::h...', 1, 'Call to undefin...', '/Users/jace/dev...', 21, Array)
#3 /Users/jace/dev/dogsquare_web/lib/Cake/Core/App.php(900): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-06-18 07:50:41 Error: [MissingViewException] View file "/Users/jace/dev/dogsquare_web/app/View/Users/create.ctp" is missing.
Exception Attributes: array (
  'file' => '/Users/jace/dev/dogsquare_web/app/View/Users/create.ctp',
)
Request URL: /users/create
Stack Trace:
#0 /Users/jace/dev/dogsquare_web/lib/Cake/View/View.php(468): View->_getViewFileName(NULL)
#1 /Users/jace/dev/dogsquare_web/lib/Cake/Controller/Controller.php(948): View->render(NULL, NULL)
#2 /Users/jace/dev/dogsquare_web/lib/Cake/Routing/Dispatcher.php(194): Controller->render()
#3 /Users/jace/dev/dogsquare_web/lib/Cake/Routing/Dispatcher.php(162): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest), Object(CakeResponse))
#4 /Users/jace/dev/dogsquare_web/app/webroot/index.php(110): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2013-06-18 08:27:51 Error: [MissingViewException] View file "/Users/jace/dev/dogsquare_web/app/View/Users/search.ctp" is missing.
Exception Attributes: array (
  'file' => '/Users/jace/dev/dogsquare_web/app/View/Users/search.ctp',
)
Request URL: /users/search
Stack Trace:
#0 /Users/jace/dev/dogsquare_web/lib/Cake/View/View.php(468): View->_getViewFileName(NULL)
#1 /Users/jace/dev/dogsquare_web/lib/Cake/Controller/Controller.php(948): View->render(NULL, NULL)
#2 /Users/jace/dev/dogsquare_web/lib/Cake/Routing/Dispatcher.php(194): Controller->render()
#3 /Users/jace/dev/dogsquare_web/lib/Cake/Routing/Dispatcher.php(162): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest), Object(CakeResponse))
#4 /Users/jace/dev/dogsquare_web/app/webroot/index.php(110): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2013-06-18 08:37:32 Error: [MissingViewException] View file "/Users/jace/dev/dogsquare_web/app/View/Users/search.ctp" is missing.
Exception Attributes: array (
  'file' => '/Users/jace/dev/dogsquare_web/app/View/Users/search.ctp',
)
Request URL: /users/search
Stack Trace:
#0 /Users/jace/dev/dogsquare_web/lib/Cake/View/View.php(468): View->_getViewFileName(NULL)
#1 /Users/jace/dev/dogsquare_web/lib/Cake/Controller/Controller.php(948): View->render(NULL, NULL)
#2 /Users/jace/dev/dogsquare_web/lib/Cake/Routing/Dispatcher.php(194): Controller->render()
#3 /Users/jace/dev/dogsquare_web/lib/Cake/Routing/Dispatcher.php(162): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest), Object(CakeResponse))
#4 /Users/jace/dev/dogsquare_web/app/webroot/index.php(110): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2013-06-18 08:37:53 Error: [PDOException] SQLSTATE[42S22]: Column not found: 1054 Unknown column 'u.fname' in 'field list'
Request URL: /users/search
Stack Trace:
#0 /Users/jace/dev/dogsquare_web/lib/Cake/Model/Datasource/DboSource.php(460): PDOStatement->execute(Array)
#1 /Users/jace/dev/dogsquare_web/lib/Cake/Model/Datasource/DboSource.php(426): DboSource->_execute('select u.fname,...', Array)
#2 /Users/jace/dev/dogsquare_web/lib/Cake/Model/Datasource/DboSource.php(669): DboSource->execute('select u.fname,...', Array, Array)
#3 /Users/jace/dev/dogsquare_web/lib/Cake/Model/Datasource/DboSource.php(539): DboSource->fetchAll('select u.fname,...')
#4 [internal function]: DboSource->query('select u.fname,...')
#5 /Users/jace/dev/dogsquare_web/lib/Cake/Model/Model.php(3061): call_user_func_array(Array, Array)
#6 /Users/jace/dev/dogsquare_web/app/Model/User.php(21): Model->query('select u.fname,...')
#7 /Users/jace/dev/dogsquare_web/app/Controller/UsersController.php(35): User->search('', '', '')
#8 [internal function]: UsersController->search()
#9 /Users/jace/dev/dogsquare_web/lib/Cake/Controller/Controller.php(486): ReflectionMethod->invokeArgs(Object(UsersController), Array)
#10 /Users/jace/dev/dogsquare_web/lib/Cake/Routing/Dispatcher.php(187): Controller->invokeAction(Object(CakeRequest))
#11 /Users/jace/dev/dogsquare_web/lib/Cake/Routing/Dispatcher.php(162): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest), Object(CakeResponse))
#12 /Users/jace/dev/dogsquare_web/app/webroot/index.php(110): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#13 {main}
2013-06-18 08:38:14 Error: [PDOException] SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'findbyid' at line 1
Request URL: /users/profile
Stack Trace:
#0 /Users/jace/dev/dogsquare_web/lib/Cake/Model/Datasource/DboSource.php(460): PDOStatement->execute(Array)
#1 /Users/jace/dev/dogsquare_web/lib/Cake/Model/Datasource/DboSource.php(426): DboSource->_execute('findbyid', Array)
#2 /Users/jace/dev/dogsquare_web/lib/Cake/Model/Datasource/DboSource.php(669): DboSource->execute('findbyid', Array, Array)
#3 /Users/jace/dev/dogsquare_web/lib/Cake/Model/Datasource/DboSource.php(612): DboSource->fetchAll('findbyid', Array, Array)
#4 /Users/jace/dev/dogsquare_web/lib/Cake/Model/Model.php(800): DboSource->query('findbyid', Array, Object(User))
#5 [internal function]: Model->__call('findbyid', Array)
#6 /Users/jace/dev/dogsquare_web/app/Controller/UsersController.php(174): User->findbyid('1')
#7 [internal function]: UsersController->profile()
#8 /Users/jace/dev/dogsquare_web/lib/Cake/Controller/Controller.php(486): ReflectionMethod->invokeArgs(Object(UsersController), Array)
#9 /Users/jace/dev/dogsquare_web/lib/Cake/Routing/Dispatcher.php(187): Controller->invokeAction(Object(CakeRequest))
#10 /Users/jace/dev/dogsquare_web/lib/Cake/Routing/Dispatcher.php(162): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest), Object(CakeResponse))
#11 /Users/jace/dev/dogsquare_web/app/webroot/index.php(110): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#12 {main}
2013-06-18 08:38:30 Error: [PDOException] SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'findbyid' at line 1
Request URL: /users/profile
Stack Trace:
#0 /Users/jace/dev/dogsquare_web/lib/Cake/Model/Datasource/DboSource.php(460): PDOStatement->execute(Array)
#1 /Users/jace/dev/dogsquare_web/lib/Cake/Model/Datasource/DboSource.php(426): DboSource->_execute('findbyid', Array)
#2 /Users/jace/dev/dogsquare_web/lib/Cake/Model/Datasource/DboSource.php(669): DboSource->execute('findbyid', Array, Array)
#3 /Users/jace/dev/dogsquare_web/lib/Cake/Model/Datasource/DboSource.php(612): DboSource->fetchAll('findbyid', Array, Array)
#4 /Users/jace/dev/dogsquare_web/lib/Cake/Model/Model.php(800): DboSource->query('findbyid', Array, Object(User))
#5 [internal function]: Model->__call('findbyid', Array)
#6 /Users/jace/dev/dogsquare_web/app/Controller/UsersController.php(174): User->findbyid('1')
#7 [internal function]: UsersController->profile()
#8 /Users/jace/dev/dogsquare_web/lib/Cake/Controller/Controller.php(486): ReflectionMethod->invokeArgs(Object(UsersController), Array)
#9 /Users/jace/dev/dogsquare_web/lib/Cake/Routing/Dispatcher.php(187): Controller->invokeAction(Object(CakeRequest))
#10 /Users/jace/dev/dogsquare_web/lib/Cake/Routing/Dispatcher.php(162): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest), Object(CakeResponse))
#11 /Users/jace/dev/dogsquare_web/app/webroot/index.php(110): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#12 {main}
2013-06-18 08:39:05 Error: [PDOException] SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'findbyid' at line 1
Request URL: /users/profile
Stack Trace:
#0 /Users/jace/dev/dogsquare_web/lib/Cake/Model/Datasource/DboSource.php(460): PDOStatement->execute(Array)
#1 /Users/jace/dev/dogsquare_web/lib/Cake/Model/Datasource/DboSource.php(426): DboSource->_execute('findbyid', Array)
#2 /Users/jace/dev/dogsquare_web/lib/Cake/Model/Datasource/DboSource.php(669): DboSource->execute('findbyid', Array, Array)
#3 /Users/jace/dev/dogsquare_web/lib/Cake/Model/Datasource/DboSource.php(612): DboSource->fetchAll('findbyid', Array, Array)
#4 /Users/jace/dev/dogsquare_web/lib/Cake/Model/Model.php(800): DboSource->query('findbyid', Array, Object(User))
#5 [internal function]: Model->__call('findbyid', Array)
#6 /Users/jace/dev/dogsquare_web/app/Controller/UsersController.php(174): User->findbyid('1')
#7 [internal function]: UsersController->profile()
#8 /Users/jace/dev/dogsquare_web/lib/Cake/Controller/Controller.php(486): ReflectionMethod->invokeArgs(Object(UsersController), Array)
#9 /Users/jace/dev/dogsquare_web/lib/Cake/Routing/Dispatcher.php(187): Controller->invokeAction(Object(CakeRequest))
#10 /Users/jace/dev/dogsquare_web/lib/Cake/Routing/Dispatcher.php(162): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest), Object(CakeResponse))
#11 /Users/jace/dev/dogsquare_web/app/webroot/index.php(110): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#12 {main}
2013-06-18 08:40:07 Error: [PDOException] SQLSTATE[42S22]: Column not found: 1054 Unknown column 'u.fname' in 'field list'
Request URL: /users/search
Stack Trace:
#0 /Users/jace/dev/dogsquare_web/lib/Cake/Model/Datasource/DboSource.php(460): PDOStatement->execute(Array)
#1 /Users/jace/dev/dogsquare_web/lib/Cake/Model/Datasource/DboSource.php(426): DboSource->_execute('select u.fname,...', Array)
#2 /Users/jace/dev/dogsquare_web/lib/Cake/Model/Datasource/DboSource.php(669): DboSource->execute('select u.fname,...', Array, Array)
#3 /Users/jace/dev/dogsquare_web/lib/Cake/Model/Datasource/DboSource.php(539): DboSource->fetchAll('select u.fname,...')
#4 [internal function]: DboSource->query('select u.fname,...')
#5 /Users/jace/dev/dogsquare_web/lib/Cake/Model/Model.php(3061): call_user_func_array(Array, Array)
#6 /Users/jace/dev/dogsquare_web/app/Model/User.php(21): Model->query('select u.fname,...')
#7 /Users/jace/dev/dogsquare_web/app/Controller/UsersController.php(35): User->search('', '', '')
#8 [internal function]: UsersController->search()
#9 /Users/jace/dev/dogsquare_web/lib/Cake/Controller/Controller.php(486): ReflectionMethod->invokeArgs(Object(UsersController), Array)
#10 /Users/jace/dev/dogsquare_web/lib/Cake/Routing/Dispatcher.php(187): Controller->invokeAction(Object(CakeRequest))
#11 /Users/jace/dev/dogsquare_web/lib/Cake/Routing/Dispatcher.php(162): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest), Object(CakeResponse))
#12 /Users/jace/dev/dogsquare_web/app/webroot/index.php(110): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#13 {main}
2013-06-18 08:42:29 Error: [PDOException] SQLSTATE[42S22]: Column not found: 1054 Unknown column 'u.fname' in 'field list'
Request URL: /users/search
Stack Trace:
#0 /Users/jace/dev/dogsquare_web/lib/Cake/Model/Datasource/DboSource.php(460): PDOStatement->execute(Array)
#1 /Users/jace/dev/dogsquare_web/lib/Cake/Model/Datasource/DboSource.php(426): DboSource->_execute('select u.fname,...', Array)
#2 /Users/jace/dev/dogsquare_web/lib/Cake/Model/Datasource/DboSource.php(669): DboSource->execute('select u.fname,...', Array, Array)
#3 /Users/jace/dev/dogsquare_web/lib/Cake/Model/Datasource/DboSource.php(539): DboSource->fetchAll('select u.fname,...')
#4 [internal function]: DboSource->query('select u.fname,...')
#5 /Users/jace/dev/dogsquare_web/lib/Cake/Model/Model.php(3061): call_user_func_array(Array, Array)
#6 /Users/jace/dev/dogsquare_web/app/Model/User.php(21): Model->query('select u.fname,...')
#7 /Users/jace/dev/dogsquare_web/app/Controller/UsersController.php(34): User->search(NULL, '')
#8 [internal function]: UsersController->search()
#9 /Users/jace/dev/dogsquare_web/lib/Cake/Controller/Controller.php(486): ReflectionMethod->invokeArgs(Object(UsersController), Array)
#10 /Users/jace/dev/dogsquare_web/lib/Cake/Routing/Dispatcher.php(187): Controller->invokeAction(Object(CakeRequest))
#11 /Users/jace/dev/dogsquare_web/lib/Cake/Routing/Dispatcher.php(162): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest), Object(CakeResponse))
#12 /Users/jace/dev/dogsquare_web/app/webroot/index.php(110): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#13 {main}
2013-06-18 08:42:53 Error: [PDOException] SQLSTATE[42S22]: Column not found: 1054 Unknown column 'u.fname' in 'field list'
Request URL: /users/search
Stack Trace:
#0 /Users/jace/dev/dogsquare_web/lib/Cake/Model/Datasource/DboSource.php(460): PDOStatement->execute(Array)
#1 /Users/jace/dev/dogsquare_web/lib/Cake/Model/Datasource/DboSource.php(426): DboSource->_execute('select u.fname,...', Array)
#2 /Users/jace/dev/dogsquare_web/lib/Cake/Model/Datasource/DboSource.php(669): DboSource->execute('select u.fname,...', Array, Array)
#3 /Users/jace/dev/dogsquare_web/lib/Cake/Model/Datasource/DboSource.php(539): DboSource->fetchAll('select u.fname,...')
#4 [internal function]: DboSource->query('select u.fname,...')
#5 /Users/jace/dev/dogsquare_web/lib/Cake/Model/Model.php(3061): call_user_func_array(Array, Array)
#6 /Users/jace/dev/dogsquare_web/app/Model/User.php(21): Model->query('select u.fname,...')
#7 /Users/jace/dev/dogsquare_web/app/Controller/UsersController.php(34): User->search(NULL, '')
#8 [internal function]: UsersController->search()
#9 /Users/jace/dev/dogsquare_web/lib/Cake/Controller/Controller.php(486): ReflectionMethod->invokeArgs(Object(UsersController), Array)
#10 /Users/jace/dev/dogsquare_web/lib/Cake/Routing/Dispatcher.php(187): Controller->invokeAction(Object(CakeRequest))
#11 /Users/jace/dev/dogsquare_web/lib/Cake/Routing/Dispatcher.php(162): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest), Object(CakeResponse))
#12 /Users/jace/dev/dogsquare_web/app/webroot/index.php(110): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#13 {main}
2013-06-18 08:44:35 Error: [PDOException] SQLSTATE[42S22]: Column not found: 1054 Unknown column 'u.fname' in 'field list'
Request URL: /users/search
Stack Trace:
#0 /Users/jace/dev/dogsquare_web/lib/Cake/Model/Datasource/DboSource.php(460): PDOStatement->execute(Array)
#1 /Users/jace/dev/dogsquare_web/lib/Cake/Model/Datasource/DboSource.php(426): DboSource->_execute('select u.fname,...', Array)
#2 /Users/jace/dev/dogsquare_web/lib/Cake/Model/Datasource/DboSource.php(669): DboSource->execute('select u.fname,...', Array, Array)
#3 /Users/jace/dev/dogsquare_web/lib/Cake/Model/Datasource/DboSource.php(539): DboSource->fetchAll('select u.fname,...')
#4 [internal function]: DboSource->query('select u.fname,...')
#5 /Users/jace/dev/dogsquare_web/lib/Cake/Model/Model.php(3061): call_user_func_array(Array, Array)
#6 /Users/jace/dev/dogsquare_web/app/Model/User.php(21): Model->query('select u.fname,...')
#7 /Users/jace/dev/dogsquare_web/app/Controller/UsersController.php(34): User->search(NULL, '')
#8 [internal function]: UsersController->search()
#9 /Users/jace/dev/dogsquare_web/lib/Cake/Controller/Controller.php(486): ReflectionMethod->invokeArgs(Object(UsersController), Array)
#10 /Users/jace/dev/dogsquare_web/lib/Cake/Routing/Dispatcher.php(187): Controller->invokeAction(Object(CakeRequest))
#11 /Users/jace/dev/dogsquare_web/lib/Cake/Routing/Dispatcher.php(162): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest), Object(CakeResponse))
#12 /Users/jace/dev/dogsquare_web/app/webroot/index.php(110): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#13 {main}
2013-06-18 08:53:42 Error: [PDOException] SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'findbyid' at line 1
Request URL: /users/edit/1
Stack Trace:
#0 /Users/jace/dev/dogsquare_web/lib/Cake/Model/Datasource/DboSource.php(460): PDOStatement->execute(Array)
#1 /Users/jace/dev/dogsquare_web/lib/Cake/Model/Datasource/DboSource.php(426): DboSource->_execute('findbyid', Array)
#2 /Users/jace/dev/dogsquare_web/lib/Cake/Model/Datasource/DboSource.php(669): DboSource->execute('findbyid', Array, Array)
#3 /Users/jace/dev/dogsquare_web/lib/Cake/Model/Datasource/DboSource.php(612): DboSource->fetchAll('findbyid', Array, Array)
#4 /Users/jace/dev/dogsquare_web/lib/Cake/Model/Model.php(800): DboSource->query('findbyid', Array, Object(User))
#5 [internal function]: Model->__call('findbyid', Array)
#6 /Users/jace/dev/dogsquare_web/app/Controller/UsersController.php(82): User->findbyid('1')
#7 [internal function]: UsersController->edit('1')
#8 /Users/jace/dev/dogsquare_web/lib/Cake/Controller/Controller.php(486): ReflectionMethod->invokeArgs(Object(UsersController), Array)
#9 /Users/jace/dev/dogsquare_web/lib/Cake/Routing/Dispatcher.php(187): Controller->invokeAction(Object(CakeRequest))
#10 /Users/jace/dev/dogsquare_web/lib/Cake/Routing/Dispatcher.php(162): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest), Object(CakeResponse))
#11 /Users/jace/dev/dogsquare_web/app/webroot/index.php(110): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#12 {main}
2013-06-18 08:55:09 Error: [PDOException] SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'findbyid' at line 1
Request URL: /users/profile
Stack Trace:
#0 /Users/jace/dev/dogsquare_web/lib/Cake/Model/Datasource/DboSource.php(460): PDOStatement->execute(Array)
#1 /Users/jace/dev/dogsquare_web/lib/Cake/Model/Datasource/DboSource.php(426): DboSource->_execute('findbyid', Array)
#2 /Users/jace/dev/dogsquare_web/lib/Cake/Model/Datasource/DboSource.php(669): DboSource->execute('findbyid', Array, Array)
#3 /Users/jace/dev/dogsquare_web/lib/Cake/Model/Datasource/DboSource.php(612): DboSource->fetchAll('findbyid', Array, Array)
#4 /Users/jace/dev/dogsquare_web/lib/Cake/Model/Model.php(800): DboSource->query('findbyid', Array, Object(User))
#5 [internal function]: Model->__call('findbyid', Array)
#6 /Users/jace/dev/dogsquare_web/app/Controller/UsersController.php(175): User->findbyid('1')
#7 [internal function]: UsersController->profile()
#8 /Users/jace/dev/dogsquare_web/lib/Cake/Controller/Controller.php(486): ReflectionMethod->invokeArgs(Object(UsersController), Array)
#9 /Users/jace/dev/dogsquare_web/lib/Cake/Routing/Dispatcher.php(187): Controller->invokeAction(Object(CakeRequest))
#10 /Users/jace/dev/dogsquare_web/lib/Cake/Routing/Dispatcher.php(162): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest), Object(CakeResponse))
#11 /Users/jace/dev/dogsquare_web/app/webroot/index.php(110): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#12 {main}
2013-06-18 09:39:01 Error: [PDOException] SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'findbyid' at line 1
Request URL: /users/edit/1
Stack Trace:
#0 /Users/jace/dev/dogsquare_web/lib/Cake/Model/Datasource/DboSource.php(460): PDOStatement->execute(Array)
#1 /Users/jace/dev/dogsquare_web/lib/Cake/Model/Datasource/DboSource.php(426): DboSource->_execute('findbyid', Array)
#2 /Users/jace/dev/dogsquare_web/lib/Cake/Model/Datasource/DboSource.php(669): DboSource->execute('findbyid', Array, Array)
#3 /Users/jace/dev/dogsquare_web/lib/Cake/Model/Datasource/DboSource.php(612): DboSource->fetchAll('findbyid', Array, Array)
#4 /Users/jace/dev/dogsquare_web/lib/Cake/Model/Model.php(800): DboSource->query('findbyid', Array, Object(User))
#5 [internal function]: Model->__call('findbyid', Array)
#6 /Users/jace/dev/dogsquare_web/app/Controller/UsersController.php(82): User->findbyid('1')
#7 [internal function]: UsersController->edit('1')
#8 /Users/jace/dev/dogsquare_web/lib/Cake/Controller/Controller.php(486): ReflectionMethod->invokeArgs(Object(UsersController), Array)
#9 /Users/jace/dev/dogsquare_web/lib/Cake/Routing/Dispatcher.php(187): Controller->invokeAction(Object(CakeRequest))
#10 /Users/jace/dev/dogsquare_web/lib/Cake/Routing/Dispatcher.php(162): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest), Object(CakeResponse))
#11 /Users/jace/dev/dogsquare_web/app/webroot/index.php(110): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#12 {main}
2013-06-18 09:41:27 Error: [PDOException] SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'findbyid' at line 1
Request URL: /users/edit/1
Stack Trace:
#0 /Users/jace/dev/dogsquare_web/lib/Cake/Model/Datasource/DboSource.php(460): PDOStatement->execute(Array)
#1 /Users/jace/dev/dogsquare_web/lib/Cake/Model/Datasource/DboSource.php(426): DboSource->_execute('findbyid', Array)
#2 /Users/jace/dev/dogsquare_web/lib/Cake/Model/Datasource/DboSource.php(669): DboSource->execute('findbyid', Array, Array)
#3 /Users/jace/dev/dogsquare_web/lib/Cake/Model/Datasource/DboSource.php(612): DboSource->fetchAll('findbyid', Array, Array)
#4 /Users/jace/dev/dogsquare_web/lib/Cake/Model/Model.php(800): DboSource->query('findbyid', Array, Object(User))
#5 [internal function]: Model->__call('findbyid', Array)
#6 /Users/jace/dev/dogsquare_web/app/Controller/UsersController.php(82): User->findbyid('1')
#7 [internal function]: UsersController->edit('1')
#8 /Users/jace/dev/dogsquare_web/lib/Cake/Controller/Controller.php(486): ReflectionMethod->invokeArgs(Object(UsersController), Array)
#9 /Users/jace/dev/dogsquare_web/lib/Cake/Routing/Dispatcher.php(187): Controller->invokeAction(Object(CakeRequest))
#10 /Users/jace/dev/dogsquare_web/lib/Cake/Routing/Dispatcher.php(162): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest), Object(CakeResponse))
#11 /Users/jace/dev/dogsquare_web/app/webroot/index.php(110): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#12 {main}
2013-06-18 11:58:10 Error: Fatal Error (1): Call to a member function create() on a non-object in [/Users/jace/dev/dogsquare_web/app/Controller/UsersController.php, line 60]
2013-06-18 11:58:10 Error: [FatalErrorException] Call to a member function create() on a non-object
Request URL: /users/create
Stack Trace:
#0 /Users/jace/dev/dogsquare_web/lib/Cake/Error/ErrorHandler.php(184): ErrorHandler::handleFatalError(1, 'Call to a membe...', '/Users/jace/dev...', 60)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', '/Users/jace/dev...', 60, Array)
#2 /Users/jace/dev/dogsquare_web/lib/Cake/Core/App.php(927): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', '/Users/jace/dev...', 60, Array)
#3 /Users/jace/dev/dogsquare_web/lib/Cake/Core/App.php(900): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-06-18 12:10:56 Error: [MissingViewException] View file "/Users/jace/dev/dogsquare_web/app/View/Users/validate_email.ctp" is missing.
Exception Attributes: array (
  'file' => '/Users/jace/dev/dogsquare_web/app/View/Users/validate_email.ctp',
)
Request URL: /users/validateEmail
Stack Trace:
#0 /Users/jace/dev/dogsquare_web/lib/Cake/View/View.php(468): View->_getViewFileName(NULL)
#1 /Users/jace/dev/dogsquare_web/lib/Cake/Controller/Controller.php(948): View->render(NULL, NULL)
#2 /Users/jace/dev/dogsquare_web/lib/Cake/Routing/Dispatcher.php(194): Controller->render()
#3 /Users/jace/dev/dogsquare_web/lib/Cake/Routing/Dispatcher.php(162): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest), Object(CakeResponse))
#4 /Users/jace/dev/dogsquare_web/app/webroot/index.php(110): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2013-06-18 12:12:32 Error: Fatal Error (1): Call to a member function Object() on a non-object in [/Users/jace/dev/dogsquare_web/app/View/Users/validate_email.ctp, line 4]
2013-06-18 12:12:32 Error: [InternalErrorException] Internal Server Error
Request URL: /users/validateEmail
Stack Trace:
#0 /Users/jace/dev/dogsquare_web/lib/Cake/Error/ErrorHandler.php(184): ErrorHandler::handleFatalError(1, 'Call to a membe...', '/Users/jace/dev...', 4)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', '/Users/jace/dev...', 4, Array)
#2 /Users/jace/dev/dogsquare_web/lib/Cake/Core/App.php(927): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', '/Users/jace/dev...', 4, Array)
#3 /Users/jace/dev/dogsquare_web/lib/Cake/Core/App.php(900): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-06-18 12:12:42 Error: Fatal Error (1): Call to a member function Object() on a non-object in [/Users/jace/dev/dogsquare_web/app/View/Users/validate_email.ctp, line 4]
2013-06-18 12:12:42 Error: [InternalErrorException] Internal Server Error
Request URL: /users/validateEmail?email=a
Stack Trace:
#0 /Users/jace/dev/dogsquare_web/lib/Cake/Error/ErrorHandler.php(184): ErrorHandler::handleFatalError(1, 'Call to a membe...', '/Users/jace/dev...', 4)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', '/Users/jace/dev...', 4, Array)
#2 /Users/jace/dev/dogsquare_web/lib/Cake/Core/App.php(927): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', '/Users/jace/dev...', 4, Array)
#3 /Users/jace/dev/dogsquare_web/lib/Cake/Core/App.php(900): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-06-18 12:16:39 Error: Fatal Error (1): Call to a member function Object() on a non-object in [/Users/jace/dev/dogsquare_web/app/View/Users/validate_email.ctp, line 4]
2013-06-18 12:16:39 Error: [InternalErrorException] Internal Server Error
Request URL: /users/validateEmail
Stack Trace:
#0 /Users/jace/dev/dogsquare_web/lib/Cake/Error/ErrorHandler.php(184): ErrorHandler::handleFatalError(1, 'Call to a membe...', '/Users/jace/dev...', 4)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', '/Users/jace/dev...', 4, Array)
#2 /Users/jace/dev/dogsquare_web/lib/Cake/Core/App.php(927): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', '/Users/jace/dev...', 4, Array)
#3 /Users/jace/dev/dogsquare_web/lib/Cake/Core/App.php(900): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-06-18 12:17:37 Error: Fatal Error (1): Call to a member function Object() on a non-object in [/Users/jace/dev/dogsquare_web/app/View/Users/validate_email.ctp, line 4]
2013-06-18 12:17:37 Error: [InternalErrorException] Internal Server Error
Request URL: /users/validateEmail?email=a
Stack Trace:
#0 /Users/jace/dev/dogsquare_web/lib/Cake/Error/ErrorHandler.php(184): ErrorHandler::handleFatalError(1, 'Call to a membe...', '/Users/jace/dev...', 4)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', '/Users/jace/dev...', 4, Array)
#2 /Users/jace/dev/dogsquare_web/lib/Cake/Core/App.php(927): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', '/Users/jace/dev...', 4, Array)
#3 /Users/jace/dev/dogsquare_web/lib/Cake/Core/App.php(900): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-06-18 12:18:40 Error: [MissingHelperException] Helper class JavascriptHelper could not be found.
Exception Attributes: array (
  'class' => 'JavascriptHelper',
  'plugin' => false,
)
Request URL: /users/validateEmail?email=a
Stack Trace:
#0 /Users/jace/dev/dogsquare_web/lib/Cake/View/View.php(884): HelperCollection->load('Javascript', Array)
#1 /Users/jace/dev/dogsquare_web/lib/Cake/View/View.php(464): View->loadHelpers()
#2 /Users/jace/dev/dogsquare_web/lib/Cake/Controller/Controller.php(948): View->render(NULL, NULL)
#3 /Users/jace/dev/dogsquare_web/lib/Cake/Routing/Dispatcher.php(194): Controller->render()
#4 /Users/jace/dev/dogsquare_web/lib/Cake/Routing/Dispatcher.php(162): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest), Object(CakeResponse))
#5 /Users/jace/dev/dogsquare_web/app/webroot/index.php(110): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#6 {main}
2013-06-18 12:19:24 Error: Fatal Error (1): Call to a member function Object() on a non-object in [/Users/jace/dev/dogsquare_web/app/View/Users/validate_email.ctp, line 4]
2013-06-18 12:19:24 Error: [InternalErrorException] Internal Server Error
Request URL: /users/validateEmail?email=a
Stack Trace:
#0 /Users/jace/dev/dogsquare_web/lib/Cake/Error/ErrorHandler.php(184): ErrorHandler::handleFatalError(1, 'Call to a membe...', '/Users/jace/dev...', 4)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', '/Users/jace/dev...', 4, Array)
#2 /Users/jace/dev/dogsquare_web/lib/Cake/Core/App.php(927): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', '/Users/jace/dev...', 4, Array)
#3 /Users/jace/dev/dogsquare_web/lib/Cake/Core/App.php(900): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-06-18 12:49:23 Error: [MissingViewException] View file "/Users/jace/dev/dogsquare_web/app/View/Users/validate_email.ctp" is missing.
Exception Attributes: array (
  'file' => '/Users/jace/dev/dogsquare_web/app/View/Users/validate_email.ctp',
)
Request URL: /users/validateEmail?email=djason%40djason.com
Stack Trace:
#0 /Users/jace/dev/dogsquare_web/lib/Cake/View/View.php(468): View->_getViewFileName(NULL)
#1 /Users/jace/dev/dogsquare_web/lib/Cake/Controller/Controller.php(948): View->render(NULL, NULL)
#2 /Users/jace/dev/dogsquare_web/lib/Cake/Routing/Dispatcher.php(194): Controller->render()
#3 /Users/jace/dev/dogsquare_web/lib/Cake/Routing/Dispatcher.php(162): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest), Object(CakeResponse))
#4 /Users/jace/dev/dogsquare_web/app/webroot/index.php(110): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2013-06-18 14:49:22 Error: [MissingControllerException] Controller class QuestionsController could not be found.
Exception Attributes: array (
  'class' => 'QuestionsController',
  'plugin' => NULL,
)
Request URL: /questions/create
Stack Trace:
#0 /Users/jace/dev/dogsquare_web/app/webroot/index.php(110): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-06-19 08:21:32 Error: [MissingViewException] View file "/Users/jace/dev/dogsquare_web/app/View/Configurations/index.ctp" is missing.
Exception Attributes: array (
  'file' => '/Users/jace/dev/dogsquare_web/app/View/Configurations/index.ctp',
)
Request URL: /configurations
Stack Trace:
#0 /Users/jace/dev/dogsquare_web/lib/Cake/View/View.php(468): View->_getViewFileName(NULL)
#1 /Users/jace/dev/dogsquare_web/lib/Cake/Controller/Controller.php(948): View->render(NULL, NULL)
#2 /Users/jace/dev/dogsquare_web/lib/Cake/Routing/Dispatcher.php(194): Controller->render()
#3 /Users/jace/dev/dogsquare_web/lib/Cake/Routing/Dispatcher.php(162): Dispatcher->_invoke(Object(ConfigurationsController), Object(CakeRequest), Object(CakeResponse))
#4 /Users/jace/dev/dogsquare_web/app/webroot/index.php(110): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2013-06-19 14:09:23 Error: Fatal Error (1): Call to a member function read() on a non-object in [/Users/jace/dev/dogsquare_web/app/Controller/AppController.php, line 39]
2013-06-19 14:09:23 Error: [FatalErrorException] Call to a member function read() on a non-object
Request URL: /configurations/breedCreate
Stack Trace:
#0 /Users/jace/dev/dogsquare_web/lib/Cake/Error/ErrorHandler.php(184): ErrorHandler::handleFatalError(1, 'Call to a membe...', '/Users/jace/dev...', 39)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', '/Users/jace/dev...', 39, Array)
#2 /Users/jace/dev/dogsquare_web/lib/Cake/Core/App.php(927): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', '/Users/jace/dev...', 39, Array)
#3 /Users/jace/dev/dogsquare_web/lib/Cake/Core/App.php(900): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-06-19 14:09:29 Error: Fatal Error (1): Call to a member function read() on a non-object in [/Users/jace/dev/dogsquare_web/app/Controller/AppController.php, line 39]
2013-06-19 14:09:29 Error: [FatalErrorException] Call to a member function read() on a non-object
Request URL: /configurations/breedCreate
Stack Trace:
#0 /Users/jace/dev/dogsquare_web/lib/Cake/Error/ErrorHandler.php(184): ErrorHandler::handleFatalError(1, 'Call to a membe...', '/Users/jace/dev...', 39)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', '/Users/jace/dev...', 39, Array)
#2 /Users/jace/dev/dogsquare_web/lib/Cake/Core/App.php(927): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', '/Users/jace/dev...', 39, Array)
#3 /Users/jace/dev/dogsquare_web/lib/Cake/Core/App.php(900): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-06-19 14:11:00 Error: Fatal Error (1): Call to a member function read() on a non-object in [/Users/jace/dev/dogsquare_web/app/Controller/AppController.php, line 39]
2013-06-19 14:11:00 Error: [FatalErrorException] Call to a member function read() on a non-object
Request URL: /configurations/index
Stack Trace:
#0 /Users/jace/dev/dogsquare_web/lib/Cake/Error/ErrorHandler.php(184): ErrorHandler::handleFatalError(1, 'Call to a membe...', '/Users/jace/dev...', 39)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', '/Users/jace/dev...', 39, Array)
#2 /Users/jace/dev/dogsquare_web/lib/Cake/Core/App.php(927): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', '/Users/jace/dev...', 39, Array)
#3 /Users/jace/dev/dogsquare_web/lib/Cake/Core/App.php(900): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-06-21 08:38:20 Error: [MissingActionException] Action ConfigurationsController::breedView() could not be found.
Exception Attributes: array (
  'controller' => 'ConfigurationsController',
  'action' => 'breedView',
)
Request URL: /configurations/breedView
Stack Trace:
#0 /Users/jace/dev/dogsquare_web/lib/Cake/Routing/Dispatcher.php(187): Controller->invokeAction(Object(CakeRequest))
#1 /Users/jace/dev/dogsquare_web/lib/Cake/Routing/Dispatcher.php(162): Dispatcher->_invoke(Object(ConfigurationsController), Object(CakeRequest), Object(CakeResponse))
#2 /Users/jace/dev/dogsquare_web/app/webroot/index.php(110): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#3 {main}
2013-06-21 11:38:13 Error: [MissingActionException] Action ConfigurationsController::breedView() could not be found.
Exception Attributes: array (
  'controller' => 'ConfigurationsController',
  'action' => 'breedView',
)
Request URL: /configurations/breedView
Stack Trace:
#0 /Users/jace/dev/dogsquare_web/lib/Cake/Routing/Dispatcher.php(187): Controller->invokeAction(Object(CakeRequest))
#1 /Users/jace/dev/dogsquare_web/lib/Cake/Routing/Dispatcher.php(162): Dispatcher->_invoke(Object(ConfigurationsController), Object(CakeRequest), Object(CakeResponse))
#2 /Users/jace/dev/dogsquare_web/app/webroot/index.php(110): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#3 {main}
2013-06-21 11:38:15 Error: [MissingActionException] Action ConfigurationsController::breedView() could not be found.
Exception Attributes: array (
  'controller' => 'ConfigurationsController',
  'action' => 'breedView',
)
Request URL: /configurations/breedView
Stack Trace:
#0 /Users/jace/dev/dogsquare_web/lib/Cake/Routing/Dispatcher.php(187): Controller->invokeAction(Object(CakeRequest))
#1 /Users/jace/dev/dogsquare_web/lib/Cake/Routing/Dispatcher.php(162): Dispatcher->_invoke(Object(ConfigurationsController), Object(CakeRequest), Object(CakeResponse))
#2 /Users/jace/dev/dogsquare_web/app/webroot/index.php(110): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#3 {main}
2013-06-21 12:11:05 Error: [MissingTableException] Table dogb_r_e_e_ds for model DogBreed was not found in datasource default.
Exception Attributes: array (
  'table' => 'dogb_r_e_e_ds',
  'class' => 'DogBreed',
  'ds' => 'default',
)
Request URL: /configurations/breedCreate
Stack Trace:
#0 /Users/jace/dev/dogsquare_web/lib/Cake/Model/Model.php(3243): Model->setSource('dogb_r_e_e_ds')
#1 /Users/jace/dev/dogsquare_web/lib/Cake/Model/Model.php(1664): Model->getDataSource()
#2 /Users/jace/dev/dogsquare_web/app/Controller/ConfigurationsController.php(46): Model->save(Array)
#3 [internal function]: ConfigurationsController->breedCreate()
#4 /Users/jace/dev/dogsquare_web/lib/Cake/Controller/Controller.php(486): ReflectionMethod->invokeArgs(Object(ConfigurationsController), Array)
#5 /Users/jace/dev/dogsquare_web/lib/Cake/Routing/Dispatcher.php(187): Controller->invokeAction(Object(CakeRequest))
#6 /Users/jace/dev/dogsquare_web/lib/Cake/Routing/Dispatcher.php(162): Dispatcher->_invoke(Object(ConfigurationsController), Object(CakeRequest), Object(CakeResponse))
#7 /Users/jace/dev/dogsquare_web/app/webroot/index.php(110): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#8 {main}
